javascript/MaterialXView/source/viewer.js

```diff
@@ -629,7 +629,15 @@ export class Material
                             }
                         }
                         let collection = materialAssign.getCollection();
+                        if (collection && collection && collection.charAt(0) == "/")
```

Comment:
What is the motivation for checking the existence of `collection` twice in this expression?

Reply:
None -- it's a cut and paste typo :). I'll fix it.